tests/e2e-playwright/tests/conftest.py

```diff
@@ -94,6 +95,70 @@ def pytest_addoption(parser: pytest.Parser) -> None:
     )
 
 
+# Dictionary to store start times of tests
+_test_start_times = {}
+
+
+def _utc_now() -> datetime.datetime:
+    return datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
minor: maybe use arrow, like we do in the rest of the codebase?

Reply:
ok